node/pegnet/txhistory.go

```diff
@@ -130,7 +134,7 @@ func (p *Pegnet) SelectTransactionHistoryActionsByHash(hash *factom.Bytes32, opt
 // SelectTransactionHistoryActionsByAddress uses the lookup table to retrieve all transactions that have
 // the specified address in either inputs or outputs
 func (p *Pegnet) SelectTransactionHistoryActionsByAddress(addr *factom.FAAddress, options HistoryQueryOptions) ([]HistoryTransaction, int, error) {
-	return p.historySelectHelper("address", addr[:], options)
+	return p.historySelectHelper("address", strings.ToUpper(hex.EncodeToString(addr[:])), options)
```

Comment:
is there a reason this is uppercase?

Reply:
SQLLite hex encode does uppercase hex, and it is string matching. So I needed uppercase hex,